apps/extension/package.json

```diff
@@ -40,7 +40,7 @@
     "@types/uuid": "^9.0.1",
```

Comment:
let's remove line-clamp from the deps above?

Reply:
Done!